app/migrations/versions/98b88b5e4fcf_.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: 98b88b5e4fcf
+Revises: 279b30132f29
+Create Date: 2021-02-15 15:05:57.430038
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '98b88b5e4fcf'
+down_revision = '279b30132f29'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index(op.f('ix_custom_attribute_filter_visibility'), 'custom_attribute', ['filter_visibility'], unique=False)
+    op.create_index(op.f('ix_custom_attribute_group_visibility'), 'custom_attribute', ['group_visibility'], unique=False)
+    op.add_column('disbursement', sa.Column('state', sa.String(), nullable=True))
+    op.drop_column('disbursement', 'is_executed')
+    op.drop_index('trgm_first_name', table_name='user')
```

Comment:
This gets rid of the search indexes, we really shouldn't drop this!

Reply:
hmmm not sure how it even got in there. so i should just swap this entire migration to a pass?